libr/anal/d/cc-avr-8

```diff
@@ -0,0 +1,26 @@
+default.cc=avr
+
+avr=cc
+cc.avr.name=avr
+cc.avr.argn=r26
```

Comment:
how does all args goes to just one register r26 ? 
probably you meant to say arg1 ?